package.json

```diff
@@ -29,12 +29,14 @@
     "@keystone-6/auth": "^6.0.0",
     "@keystone-6/core": "^4.0.1",
     "@keystone-6/fields-document": "^6.0.0",
+    "@types/is-url": "^1.2.30",
     "cypress": "^11.2.0",
     "cypress-audit": "^1.1.0",
     "date-fns": "^2.29.3",
     "dotenv": "^16.0.3",
-    "typescript": "^4.9.4",
-    "node-notifier": "^10.0.1"
+    "is-url": "^1.2.4",
```

Comment:
Should the new dependency be in the UI deps only? Instead of root

Reply:
my mistake, moved